templates/app/env.hbs

```diff
@@ -7,6 +7,8 @@ DATABASE_URL={{ databaseUrl }}
 DATABASE_SCHEMA={{ dbSchema }}
 {{/if}}
 DATABASE_SSL={{ ssl }}
+# This should be removed in production environment.
+DATABASE_REJECT_UNAUTHORIZED=false
```

Comment:
[MISC] Shouldn't this env var be added only when `DATABASE_SSL` is `true`?
As discussed, not a big fan of `REJECT_UNAUTHORIZED=false` by default (Especially when a user ask specifically for SSL), but I guess this has been discussed

Reply:
> Shouldn't this env var be added only when DATABASE_SSL is true?

You may be right. Still, assuming users could switch `DATABASE_SSL` to `true/false`, they also should be able to change this options (or at least to be aware it exists). 

> As discussed, not a big fan of REJECT_UNAUTHORIZED=false by default (Especially when a user ask specifically for SSL), but I guess this has been discussed

Yes, it has been done by design. It is in `.env` so the developer will not miss it when creating a new environment (eg: prod, staging). So I guess it's OK.